video/out/opengl/common.c

```diff
@@ -274,6 +274,23 @@ static const struct gl_functions gl_functions[] = {
         },
     },
     {
+        .ver_core = 320,
+        .extension = "GL_ARB_timer_query",
+        .functions = (const struct gl_function[]) {
+            DEF_FN(GenQueries),
+            DEF_FN(DeleteQueries),
+            DEF_FN(BeginQuery),
+            DEF_FN(EndQuery),
+            DEF_FN(QueryCounter),
+            DEF_FN(IsQuery),
+            DEF_FN(GetQueryObjectiv),
+            DEF_FN(GetQueryObjecti64v),
+            DEF_FN(GetQueryObjectuiv),
+            DEF_FN(GetQueryObjectui64v),
```

Comment:
The compatibility matrix on opengl.org says 3.2, the description says 3.3. I don't know which one is accurate. Maybe we should just set it to 3.3 to be sure?

(Note: I copied these almost verbatim from one of your commits, which also used 3.2 here)